source/Directory.Build.props

```diff
@@ -26,4 +26,8 @@
     <WarningsNotAsErrors>$(WarningsNotAsErrors);CS0612;CS0618;CS1030</WarningsNotAsErrors>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="DotNet.ReproducibleBuilds" Version="1.1.1" PrivateAssets="All"/>
```

Comment:
`style` _non-blocking_
Would you mind adding a space before the closing tag for consistency?
```diff
-...PrivateAssets="All"/>
+...PrivateAssets="All" />
```